### ISSUE TITLE
Writing the "Contributing rules" for the handbook

### ISSUE THREAD

**Issue by Nirus2000** [portfolio-performance/portfolio-help#16]
A developer is needed to create the default rule.
It is about formats, video resolutions, writing, image formats, structure, content overlays etc.

The "[Contributing rules](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md)" from Portfolio Performance can serve as a template here.

**Comment by hug-sch** [comment]
Hi Nirus2000, do you mean something like a style guide? There are quite a few mentioned by [Writethedocs.org](https://www.writethedocs.org/guide/writing/style-guides/). 

I'm currently trying to update the PP manual.  I could try to make a condensed and appropriate (for us) version of one of those styleguides mentioned above.

**Comment by Nirus2000** [comment]
Hello @hug-sch 
Yes, that is exactly what we are looking for.
We would like to set a standard so that the manual is created uniformly. This is about video formats, light instead of dark theme, organization and structure, graphics and so on.

We would be happy if you could support us here.

Regards
Alex

**Comment by hug-sch** [comment]
Hello @Nirus2000 
I have started creating the style guide. You can find a beginning of it at CONTRIBUTING.md. I will regularly add to it while writing the PP manual. Please, feel free to comment.

**Comment by Nirus2000** [comment]
Hello @hug-sch 
starting a pull request to work together... 👍🏻

**Comment by flywire** [comment]
> The project uses a https://github.com/squidfunk/mkdocs-material/discussions/2346:

Can the rules explain how to contribute different languages. For example, Is the content of the different languages a translation or is the content independent? Is there a main language with all content?

**Comment by flywire** [comment]
https://github.com/portfolio-performance/portfolio-help/blob/master/CONTRIBUTING.md#writing-setup
> At present, the manual is available in two languages: English (en) and German (de). 

I suggest inserting: `Each language is a separate version and contributors translate the content manually.`

> All documentation pages should be stored within these directories; including images and assets (e.g. demo-portfolio.xml).

What about shared content? See https://github.com/squidfunk/mkdocs-material/discussions/2346#discussion-3240838

**Comment by hug-sch** [comment]
> I suggest inserting: `Each language is a separate version and contributors translate the content manually.`

I agree. Have updated the contributing.md following your suggestion.
 
> What about shared content? See [squidfunk/mkdocs-material#2346 (comment)](https://github.com/squidfunk/mkdocs-material/discussions/2346#discussion-3240838)

The link above describes a possible solution. I'm not sure there will be many translations or extensive need for shared content between the language modules.

**Comment by flywire** [comment]
I'd emphasize markup is text, so concentrate on the content before the (simple) syntax to format it.

> Go to [GitHub](https://github.com/) and click on Sign up.

Why? Only do that if GitHub prompts for it.

---

https://github.com/portfolio-performance/portfolio-help#contributing seems overly complex. Perhaps something like https://dokuwiki.librecad.org/doku.php/usage:faq#can_i_easily_contribute_on_github would be simpler and adequate for most contributions?

**Comment by Nirus2000** [comment]
#130
